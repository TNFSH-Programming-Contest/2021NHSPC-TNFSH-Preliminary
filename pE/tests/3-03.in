23 50
3 4 12
3 4 22
5 3 10
6 7 22
5 4 7
5 4 6
5 4 7
7 8 25
5 4 17
9 10 12
11 12 26
11 12 26
13 10 29
8 4 27
5 6 23
13 10 19
14 5 18
15 13 19
11 16 26
4 17 16
12 9 16
18 2 5
16 15 15
17 11 11
11 10 21
11 10 6
14 5 28
5 17 28
14 5 28
11 10 18
17 11 23
19 20 19
18 2 26
1 21 13
1 21 28
20 18 12
14 11 24
1 22 22
1 22 7
20 2 14
1 22 9
21 14 14
1 22 11
19 20 13
10 19 20
23 21 22
23 21 19
1 23 12
21 14 11
22 2 29
