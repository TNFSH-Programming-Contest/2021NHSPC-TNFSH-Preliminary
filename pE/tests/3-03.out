21.0821116624706765084
